torch/onnx/_internal/diagnostics/infra/context.py

```diff
@@ -17,6 +17,7 @@
     Literal,
     Mapping,
     Optional,
+    Self,
```

Comment:
We support down to Python 3.8 so this import will need to come from the typing_extensions backport.

Reply:
Thanks for catching this! Import updated.